docs/getting-started.md

```diff
@@ -199,7 +199,22 @@ You can view restore statues with `ListRestoreStatuses`.
 $ grpcurl -plaintext -d '{"limit": 3, "cluster_id": "CLUSTER-ID"}' 192.168.0.254:20051 rpc.Cassy.ListRestoreStatuses
 ```
 
-## Future Work
+### Scheduling backups
 
-The following features are planned to be implemented in the near future.
-* GUI component to make backup and restore operations even easier. 
+You can schedule backups using the `cassy-schedule` executable together with `cron`. 
+
+The command can be executed with `cassy-schedule [-t=<timeout>] [subcommand] [options]`. The `timeout` option is an integer in seconds, which will set a maximum time to wait for the backup to reach a `COMPLETED` or `FAILED` status, or otherwise exit the program with an exception. If you do not set this option it will use the default value of 20.
+
+The subcommands are used for setting the backup type, and can be used as follows:
+
+- `cluster_snapshot [-c, --cluster_id]` : take a cluster-wide backup
+- `node_snapshot [-c, --cluster_id] [-i, --target_ips]` : take a node snapshot
+- `node_incremental [-c, --cluster_id] [-i, --target_ips` : take a node incremental backup using the most recent successful snapshot.
```

Comment:
I don't think incremental backup always works for the previously successful snapshot if the latest one fails because an incremental backup is additions to the latest snapshot.

Reply:
The reason I worded it this way is because there is a check in the code to make sure we ignore snapshots that are not `COMPLETED`, so we only ever take incremental backups of successful snapshots.

Reply:
We don't know how the previous snapshot is not `COMPLETED` so there might be a case where a snapshot is properly taken and incremental backups are all removed but failed to report back to the master. In that case, remained incremental backups are not sufficient since there is no data between the last successful snapshot and the last failed snapshot.

Reply:
@feeblefakie 
So you are suggesting the process should be like this? I'm sorry, I'm still a little confused about the business logic here and also in the comment above about the multi-node cluster.

```
// get recent snapshot
if (snapshot is COMPLETED) {
    take incremental backup
}
else {
    take new snapshot
    // wait for COMPLETED status
    if (new snapshot is FAILED) {
        ??
    }
    take incremental snapshot
}

Reply:
@yume-ind I think it can throw an exception if the specified snapshot is not completed.